bikestreets-ios/Map/MapsViewController.swift

```diff
@@ -130,9 +132,10 @@ class MapsViewController: UIViewController {
   // MARK: -- Layer Placement
   
   internal var belowRoadLabelLayer: LayerPosition? {
-    let id = "road-label"
-    let roadLabelId =  mapView.mapboxMap.style.styleManager.styleLayerExists(forLayerId: id) ? id : nil
-    
+    // Either "road-label" or "road-label-simple" could be used as the road label layer
+    let roadLabelPrefix = "road-label"
+    let roadLabelLayer = mapView.mapboxMap.style.styleManager.getStyleLayers().filter({ $0.id.hasPrefix(roadLabelPrefix) }).first ?? nil
+    let roadLabelId =  roadLabelLayer?.id ?? nil
```

Comment:
Can we use [`first(where:)`](https://developer.apple.com/documentation/swift/array/first(where:)) instead?

```suggestion
    let roadLabelLayer = mapView.mapboxMap.style.styleManager.getStyleLayers().first(where: { $0.id.hasPrefix(roadLabelPrefix) })
    let roadLabelId =  roadLabelLayer?.id
```

Reply:
Good call